catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/sierra/SierraLocation.scala

```diff
@@ -25,4 +25,17 @@ trait SierraLocation {
         )
       case None => None
     }
+
+  def getDigitalLocation(identifier: String): DigitalLocation = {
+    if (!identifier.isEmpty) {
```

Comment:
Yes I see, maybe it can't be.  I didn't like passing it through if it's clearly invalid though.

Reply:
👍 

Maybe a comment that this is being defensive rather than in response to specific data? And we can point to specific examples if/when we throw the error.